Sources/Web3Core/Transaction/CodableTransaction.swift

```diff
@@ -219,53 +219,53 @@ extension CodableTransaction: Codable {
 
     public func encode(to encoder: Encoder) throws {
         // FIXME: There's a huge mess here, please take a look here at code review if any.
-        var containier = encoder.container(keyedBy: CodingKeys.self)
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.